packages/hash/frontend/src/pages/[account-slug]/types/entity-type/[entity-type-id].page.tsx

```diff
@@ -167,14 +169,17 @@ const Page: NextPageWithLayout = () => {
       console.warn(
         `Error: Couldn't find entity type with id '${router.query["entity-type-id"]}'.`,
       );
-      void router.replace(`/@${namespace?.shortname}/types/new/entity-type`);
+      void router.replace(
+        `/@${activeWorkspace?.shortname}/types/new/entity-type`,
+      );
```

Comment:
@nathggns I'm not sure I've understood this correctly, why would we want to redirect to the create entity type page here if the `namespace` couldn't be found from the URL? Wouldn't we want to display a 404 instead?

Reply:
https://github.com/hashintel/hash/pull/1527 changes this behavior

Reply:
Ah great, thanks for flagging Ciaran :)